hello-app/cloudshell/tutorial.md

```diff
@@ -0,0 +1,133 @@
+gcloud config set compute/zone COMPUTE_ZONE
+```    
+Replace COMPUTE_ZONE with your [compute zone](https://cloud.google.com/compute/docs/regions-zones#available), such as `us-west1-a`.
+
+
+## Create a GKE cluster
+A cluster consists of at least one cluster control plane machine and multiple worker machines called nodes. Nodes are [Compute Engine virtual machine (VM) instances](https://cloud.google.com/compute/docs/instances) that run the Kubernetes processes necessary to make them part of the cluster. 
+
+GKE offers two [modes of operation](https://cloud.google.com/kubernetes-engine/docs/concepts/types-of-clusters#modes) for clusters: Standard and Autopilot. For this tutorial, we'll use Standard mode.
+
+### 1. Create a Standard GKE cluster
+
+Run the command below in your terminal to create a one-node Standard cluster named `hello-cluster`:
+```
+gcloud container clusters create hello-cluster --num-nodes=1
+```
+
+It might take several minutes to finish creating the cluster.
+
+
+### 2. Get authentication credentials
+
+After creating your cluster, you need to get authentication credentials to interact with the cluster.
+
+```
+gcloud container clusters get-credentials hello-cluster
+```
+
+This command configures `kubectl` to use the cluster you created.    
+
+
+Next, let's deploy an app to the cluster.
+
+## Deploy an application to the cluster
+
+Now that you have created a cluster, you can deploy a [containerized application](https://cloud.google.com/kubernetes-engine/docs/concepts/kubernetes-engine-overview#workloads) to it. For this tutorial, let's deploy the example web application `hello-app`.
+
+GKE uses Kubernetes objects to create and manage your cluster's resources. This example has two types of Kubernetes objects:
+- a [Deployment](https://cloud.google.com/kubernetes-engine/docs/concepts/deployment) object, which deploys stateless applications like web servers 
+- a [Service](https://cloud.google.com/kubernetes-engine/docs/concepts/service) object, which defines rules and load balancing for accessing your application from the internet
+
+### Create the Deployment
+
+To run `hello-app` in your cluster, run the following command:
+```
+kubectl create deployment hello-server --image=gcr.io/google-samples/hello-app:1.0
+```
+
+Let's break down what this command is doing:
+- The Kubernetes command [`kubectl create deployment`](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#create) creates a Deployment named `hello-server`. The Deployment's [Pod](https://cloud.google.com/kubernetes-engine/docs/concepts/pod) runs the `hello-app` container image.
+
+- The command flag `--image` specifies a container image to deploy. In this case, the command pulls the example image from a [Container Registry](https://cloud.google.com/container-registry/docs) bucket, `gcr.io/google-samples/hello-app`. The suffix `:1.0` indicates the specific image version to pull. (If you don't specify a version, the latest version is used.)
```

Comment:
Can we also mention that artifact registry is an option and add a link? https://cloud.google.com/artifact-registry/docs

This product is the new version of container registry (may eventually replace gcr as the default for samples in the future)

Reply:
@peterlin741 After going through Artifact Registry docs, I'm now thinking we should omit GCR entirely here, and have the command deploy to AR. (There are already public AR images available for the k8s engine samples, i.e. `us-docker.pkg.dev/google-samples/containers/gke/hello-app`.)

Thoughts?

Reply:
As of now, we're still considering this, but leaning towards sticking to GCR for the default for samples, since AR introduces quite a bit of friction. I'll add you to the discussion docs, once finalized

Reply:
In that case, let's leave this section as-is for now and we can refactor the tutorial to use AR once the direction on that is clear. (This section is a big chunk of words and I'm hesitant to make it even wordier.)